Gemfile

```diff
@@ -2,6 +2,7 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'mime-types', '~> 2.99' if RUBY_VERSION < '2.0'
```

Comment:
Good idea!

We can probably just depend on mime types being `< 3`, and we'll want to make sure that the ruby version isn't a string comparison. I'll merge with a couple of extra commits.